### ISSUE TITLE
Screen Reader does not announce the state of 'expand (-) and Collapse (+) icon buttons.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#103]
User Experience:

Current state of the element is an important information for the user to know and move further with the process, screen reader users will get impacted if state information is not provided via screen reader.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.​
3. Wait portal to load in design mode​
4. Navigate to the Left nav and select Navigation button.​
5. Enable screen reader and navigate to the 'expand (-) and Collapse (+) icon buttons.
6. Verify whether screen reader announcing state or not.
 

Actual Result:

Screen Reader does not announce the state of 'expand (-) and Collapse (+) icon buttons, present under 'Navigation' popup.

 

Expected Result:

Screen Reader should  announce the state as 'expanded and collapsed' respectively for 'expand (-) and Collapsed (+) icon buttons, present under 'Navigation' popup.

 

Note:

Similar issue exist with Chrome + JAWS and NVDA.

Similar issue exist with this type of buttons present in application

 

MAS Reference: 

MAS 4.1.2 - Name, Role, Value (40B)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make      bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries

**Comment by azaslonov** [comment]
Fixed.
